scripts/configure-installer-tests-cluster.sh

```diff
@@ -13,8 +13,13 @@ KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_D
 # Default values
 OC_STABLE_LOGIN="false"
 CI_OPERATOR_HUB_PROJECT="ci-operator-hub-project"
+# Copy kubeconfig to temporary kubeconfig file
+# Read, Write and Execute permission to temporary kubeconfig file
+mkdir -p ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
+cp ${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
+chmod 764 ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
```

Comment:
why 764?

Reply:
Hmm, I gave unnecessary write permission to the file. Read and write permission would be enough for kubeconfig file (666 ?). Anyway let me check what is the default permission of kubeconfig file used in template based test infra.

Reply:
$ ls -l
total 32
-rw-r--r--@ 1 amit  staff     23 Jun 14 20:18 kubeadmin-password
-rw-r--r--@ 1 amit  staff  12193 Jun 14 20:18 kubeconfig

So it would be ```644```. Try this

Reply:
Thanks @amitkrout I will add 644 permission for the file 🙂

Reply:
Done